### PR TITLE
Change default devise id attribute

### PR DESCRIPTION
**Pull request by AW-UC**
This changes the default value of 'identificationAttributeName', as discussed in #441. It also adapts and expands the Devise-README accordingly.

### CODE REVIEW

**Comment by AW-UC** [comment]
@marcoow please review.
now you will only have to adapt the API docs:
http://ember-simple-auth.com/ember-simple-auth-devise-api-docs.html

File: packages/ember-simple-auth-devise/README.md [hunk 1]
```diff
@@ -63,8 +90,8 @@ class SessionsController < Devise::SessionsController
     super do |user|
       if request.format.json?
         data = {
-          token:      user.authentication_token,
-          user_email: user.email
+          token:  user.authentication_token,
+          user:   user.email
```
Comment (marcoow) [inline packages/ember-simple-auth-devise/README.md]: this must be `email`

**Comment by marcoow** [comment]
The API docs are updated as part of the release process.

**Comment by AW-UC** [comment]
fixed typo.

**Pull request merged**